pkg/web/templates/layouts/sunrise.html

```diff
@@ -0,0 +1,19 @@
+{{ define "sunrise" }}
+<!DOCTYPE html>
+<html lang="en">
+  {{ template "head" . }}
+<body>
+
+  {{ template "sunrise_header" . }}
+  
+  <main>
+    {{ block "content" . }}{{ end }}
+  </main>
+  <script src="https://unpkg.com/htmx.org@1.9.10" integrity="sha384-D1Kt99CQMDuVetoL1lrYwg5t+9QdHe7NLX/SoJYkXDFfX37iInKRy5xLSi8nO7UC" crossorigin="anonymous"></script>
```

Comment:
I'll create a follow on story to move the scripts used in each layout into a separate reusable component.